test/hosting/NuGet.Config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="dotnet5-rtm" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/5.0.100-rtm.20526.5/nuget/v3/index.json" />
```

Comment:
Do we still need this feed given that .NET 5 GA is out? We should be able to get all packages from nuget.org now.

Reply:
Aditya needs to clean this up in several places, we will do another PR